app/components/createService/createservice.html

```diff
@@ -316,7 +317,55 @@
             </form>
           </div>
           <!-- !tab-labels -->
+          <!-- tab-security -->
```

Comment:
I do like what you did here with the help text, but we need to apply some css on it as it's not really aligned with the labels/inputs.

![capture du 2016-12-25 09 06 30](https://cloud.githubusercontent.com/assets/5485061/21468643/c2fc6df8-ca81-11e6-975d-5b76802e7010.png)

`margin-top: 10px;` on the `<p>` elements should be enough.